### PR TITLE
chore(quickstarts): Adding readme to the gradle webapp quickstart and example how to override webapp generator defaults with gradle DSL

### PR DESCRIPTION
**Pull request by sunix**
## Description
Adding readme to the gradle webapp quickstart and example how to override webapp generator defaults with gradle DSL

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by manusa — APPROVED** [review]

File: quickstarts/gradle/webapp/build.gradle [hunk 1]
```diff
@@ -26,3 +26,15 @@ compileJava {
 compileTestJava {
     options.encoding = 'UTF-8'
 }
+
+/*
+kubernetes {
```
Comment (rohanKanojia) [inline quickstarts/gradle/webapp/build.gradle:31]: I was wondering if we could enable this block based on some property:
```groovy
if (project.hasProperty('enableJetty')) {
kubernetes {
    generator {
        config {
            'webapp' {
                server = "jetty"
            }
        }
    }
}
}
```

if we do this, the user won't need to go to `build.gradle` and comment/uncomment to switch between tomcat and jetty. Default behavior would be with tomcat and `gradle k8sBuild -PenableJetty` would enable jetty.

**Review by rohanKanojia — APPROVED** [review]

File: quickstarts/gradle/webapp/build.gradle (hunk 1 above)
Comment (sunix) [reply quickstarts/gradle/webapp/build.gradle:31]: I understand, but I would like to force the user to go and read/modify the build.gradle file if he which to modify it and try jetty (or wildfly) in a learning purpose. Does it make sense?

Comment (rohanKanojia) [reply quickstarts/gradle/webapp/build.gradle:31]: sure, I'm totally fine with this.

**Pull request merged**